pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -387,6 +387,11 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 					Resources: util.NewStringSet("services"),
 				},
 				{
+					// Needed to get namespace<->id mappings for service isolation rules
```

Comment:
if this is for SDN, shouldn't it be in the "system:sdn-reader" role? nodes are already included in the "system:sdn-reader" role, so they already have permission to get/list/watch namespaces